src/sentry/api/helpers/group_index.py

```diff
@@ -87,17 +87,11 @@ def build_query_params_from_request(request, projects, environments):
                 request.user,
                 environments,
             )
-        except Exception:
-            # TODO: Catch less broad exceptions when we're confident in these
-            # new filters
-            logging.exception('Error occurred while parsing new style search query')
-            search_filters = []
-            # If something goes wrong here we just want to use the working
-            # filters
-            use_new_filters = False
+        except InvalidSearchQuery as e:
+            raise ValidationError('Your search query could not be parsed: {}'.format(e.message))
```

Comment:
```suggestion
            raise ValidationError(u'Your search query could not be parsed: {}'.format(e.message))
```
UnicodeError :ghost: